src/app/shared/issue-tables/issue-sorter.ts

```diff
@@ -1,8 +1,8 @@
-import { MatSort } from '@angular/material/sort';
+import { Sort } from '@angular/material/sort';
 import * as moment from 'moment';
 import { Issue } from '../../core/models/issue.model';
 
-export function getSortedData(sort: MatSort, data: Issue[]): Issue[] {
+export function getSortedData(sort: Sort, data: Issue[]): Issue[] {
```

Comment:
Perhaps we can rename this to `applySort`, in the same convention as applying filters in `./dropdownfilter.ts`?